.github/pull_request_template.md

```diff
@@ -0,0 +1,84 @@
+
+
+### What Does This PR Do?
+
+<!--
+Add detailed discussion of changes here
+This is likely a summary, or the complete contents, of your commit messages.
+-->
+
+
+### What Should Reviewers Watch For?
+
+If you're reviewing this PR, please check these things, in particular:
+
+* TODO
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply for some reason.
+
+Common items include:
+* Is this likely to address the goals expressed in the user story?
+* Are any additional documentation updates needed?
+* Are there any unhandled and/or untested edge cases you can think of?
+* Is user input properly sanitized & handled?
+* Does this make any backwards-incompatible changes that might break end user clients?
+* Can you find any bugs if you run the code locally and test it manually?
+-->
+
+
+### What Security Implications Does This PR Have?
+
+Submitters should complete the following questionnaire:
+
+* If the answer to any of these questions is **Yes**, then here's a link to the associated Security Impact Assessment: N/A.
+    * Does this PR add any new software dependencies? **Yes** or **No**.
+    * Does this PR modify or invalidate any of our security controls? **Yes** or **No**.
+    * Does this PR store or transmit data that was not stored or transmitted before? **Yes** or **No**.
+* If the answer to any of these questions is **Yes**, then please add @StewGoin as a reviewer, and note that this PR should not be merged unless/until he also approves it.
+    * Do you think this PR requires additional review of its security implications for other reasons? **Yes** or **No**.
+
+
+### What Needs to Be Deployed Before this PR?
+
+This PR cannot be deployed until the following pre-requisite changes have been fully deployed:
+
+* CMSgov/some_repo#42
+
+<!--
+Add some items to the list above, or remove the entire section if it doesn't apply.
+
+Common items include:
+* Database migrations (which should always go out in PRs by themselves, to reduce risk).
+* New features in external dependencies (e.g. BFD).
+-->
+
+
+### Submitter Checklist
+
+Before requesting final review, I've gone through and verified that this PR complies with the following requirements:
+
+* [ ] I've verified that this PR is as small as it can reasonably be, in order to:
+    1. Ensure that any problems it causes have a small "blast radius".
+    2. Ensure that it'll be easier to rollback if that becomes necessary.
+    3. Ease the burden on reviewers.
+* [ ] I've verified that the individual commits in this PR all:
+    1. Reasonably explain the "what" and "why" of the changes.
+    2. Leverage JIRA's [smart commits](https://confluence.atlassian.com/jirasoftwarecloud/processing-issues-with-smart-commits-788960027.html) to reference the JIRA ticket that they're associated with.
+* [ ] I've verified that this PR includes all required documentation changes, including changelog / release notes entries.
```

Comment:
Release note entries?  In BCDA's current process, the PR is self-documenting... in other words, the release notes are generated from the list of PRs included in the release.

Reply:
True for AB2D as well (we're using `gren`).

Reply:
Then you're okay?

Reply:
Can we please add something about README updates consistent with the change? How about `this PR includes all required documentation changes, including README updates and changelog / release notes entries.`

Reply:
Excellent suggestion, Deirdre!

Also, as a complete aside, are you familiar with GitHub's new "suggestion" feature? See #6 here: <https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request>. Also worth mentioning that GH also now supports multi-line comments and suggestions: you click and drag on the "+" instead of just single-clicking. (Apologies if you were already familiar with these -- just trying to spread the good news!)